### ISSUE TITLE
Express compound het for single variant

### ISSUE THREAD

**Issue by Relequestual** [ga4gh/mme-apis#162]
Previously issue #89 looked to make it possible to express compound het variation, however I did not fully appreciate our (Decipher's) situation.

The plan now is to revert #103, as it does not fulfil our requirements, (and was neve included in the release anyway).

In Decipher's data, the DDD Research variants only provide the individual variants per record, but they may be annotated as a compound het, without any ability (to general users) to identify or know for sure about the other variant that forms the compound het.

Essentially, I think what I was asking is a way to represent the fact that, for any given variant, it forms a compound het, but without nesecerialy specifying that other variant. This is still useful in terms of when the gene has a high Haploinsufficiency Score (e.g. 0-10%) and therfore more likely to exhibit haploinsufficiency.

Later, we may wish to express multiple variants forming a compound het. It is probable that in enabling the ability to mark a single variant as a compound het, the matching variant should be clear, if provied.

If any other system holds or shares compound het variation over MME, they should speak up regarding this change in terms of how they store their data.

It's noted that most nodes in the MME network only share gene level information in most cases.

**Comment by Relequestual** [comment]
@jxchong Right, I understand. Thanks for the info. Will try to remember this for the Case Discovery API!

I think for DECIPHER, once we support variant grouping, and flagging of compound het, it will be stored as two variants, however we aren't yet clear if both variants must have the same shared status. It's possible that one might not be shared publically even if the patient is consented.

My feeling is this is unlikely, but it IS a current case for the DDD Research Variants (which are exposed over MME).

**Comment by sj00coder** [comment]
Sir,
I want to contribute to the project.But I am not much experienced.I have skills in html,css,javascript,nodejs,react,express,mongodb,python.
please propose me some good first issues and guide me to setup the environment. 
Please guide me sir.

**Comment by Relequestual** [comment]
@sj00coder Hello. Unfortunately, this is not a project where just anyone can contribute.
This project is a specification defining a format and a process between specific services that store genetic data.
Please see matchmakerexchange.org/ for more information.

**Comment by sj00coder** [comment]
> @sj00coder Hello. Unfortunately, this is not a project where just anyone can contribute.
> This project is a specification defining a format and a process between specific services that store genetic data.
> Please see matchmakerexchange.org/ for more information.

Thank you so much for the reply.
Actually , I want to contribute for google summer of code.Can you please suggest me any  project of the organisation for the same.
Thank you once again.
